charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -59,6 +59,12 @@ rules:
   - leases
   verbs:
   - '*'
+- apiGroups:
+  - certificates.k8s.io
+  resources:
+  - certificatesigningrequests
+  verbs:
+  - '*'
```

Comment:
Can we limit this to only read operations?
/invite @mvladev

Reply:
Why? Which one of the above resources is limited to only read operations?

Reply:
My concern was about privilege escalation, but admins can already do this via the (Clusterr)Role(Binding) resources and service accounts. 
Excuse me for the inconvenience.